app/webpacker/components/Tickets/SkateholderSelector.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, Dropdown } from 'semantic-ui-react';
+import useInputState from '../../lib/hooks/useInputState';
+
+export default function SkateholderSelector({ stakeholderList, setUserSelectedStakeholder }) {
+  const [selectedOption, setSelectedOption] = useInputState(stakeholderList[0]);
+  return (
+    <>
+      <p>
+        You are part of more than one stakeholder, please select the stakeholder as which you want
+        to visit the ticket page.
+      </p>
+      <Dropdown
+        options={stakeholderList.map((requesterStakeholder) => ({
+          key: requesterStakeholder.id,
+          text: `${requesterStakeholder.stakeholder.name} (${requesterStakeholder.stakeholder_role})`,
+          value: requesterStakeholder,
+        }))}
```

Comment:
Since they are derived from the `stakeholderList` prop, it's probably safe to `memo` these. Computing them inline will only induce unnecessary code execution during rerender.